svc/GridExportService.js

```diff
@@ -381,6 +386,23 @@ export class GridExportService extends HoistService {
             fn();
         };
     }
+
+    // Return a value's data type if different from the type specified
+    getTypeIfDifferent(v, type) {
+        const ifTypeNot = (...types) => types.includes(type) ? null : last(types),
+            {BOOL, INT, NUMBER, LOCAL_DATE, DATE, STRING, PWD, AUTO, TAGS, JSON} = FieldType;
+
+        if (isBoolean(v))   return ifTypeNot(BOOL);
+        if (isInteger(v))   return ifTypeNot(NUMBER, INT);
+        if (isNumber(v))    return ifTypeNot(NUMBER);
+        if (isLocalDate(v)) return ifTypeNot(LOCAL_DATE);
+        if (isDate(v))      return ifTypeNot(DATE);
+        if (isString(v))    return ifTypeNot(PWD, STRING, AUTO);
+        if (isArray(v))     return ifTypeNot(TAGS, JSON, AUTO);
+        if (isObject(v))    return ifTypeNot(JSON, AUTO);
+
+        return null;
+    }
```

Comment:
I'm not clear on the circumstances when a cell's type would be different from the store field type? Isn't this only really possible if the store field type is AUTO?